Bottom: fecd51ab335d6bb401460c881caa4054507c16dd
Top:    f56e5b77382b15cac9bf50c08d4c9a8976aab5c8
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-31 15:58:42 -0500

Create raw repositories for ptxdist builds



---
diff --git a/ansible-playbook/group_vars/all/nexus3.yml b/ansible-playbook/group_vars/all/nexus3.yml
index 064c2bbb..06de47bd 100644
--- a/ansible-playbook/group_vars/all/nexus3.yml
+++ b/ansible-playbook/group_vars/all/nexus3.yml
@@ -113,20 +113,38 @@ nexus_repos_raw_hosted:
   - name: download
     strict_content_validation: false
     write_policy: allow  # one of "allow", "allow_once" or "deny"
-  - name: ntel-ipkg
+  - name: ntel
     strict_content_validation: false
     write_policy: allow  # one of "allow", "allow_once" or "deny"
-  - name: ntel-sstate
+  - name: bitbake-sstate
     strict_content_validation: false
     write_policy: allow  # one of "allow", "allow_once" or "deny"
-  - name: ptxdist-ipkg
+  - name: orphans
+    strict_content_validation: false
+    write_policy: allow  # one of "allow", "allow_once" or "deny"
+  - name: ptxdist
     strict_content_validation: false
     write_policy: allow  # one of "allow", "allow_once" or "deny"
 
 nexus_repos_raw_proxy:
+  - name: debian
+    remote_url: 'http://ftp.us.debian.org/debian/'
+    strict_content_validation: false
+  - name: gnu
+    remote_url: 'http://mirror.team-cymru.com/gnu/'
+    strict_content_validation: false
+  - name: kernel
+    remote_url: 'https://mirrors.edge.kernel.org/pub/'
+    strict_content_validation: false
+  - name: sourceforge
+    remote_url: 'http://downloads.sourceforge.net/project'
+    strict_content_validation: false
   - name: ubuntu-archive
     remote_url: 'http://archive.ubuntu.com/ubuntu/'
     strict_content_validation: false
+  - name: xorg
+    remote_url: 'http://xorg.mirrors.pair.com/'
+    strict_content_validation: false
 
 nexus_repos_raw_group: []
